gapic/src/main/com/google/gapid/Main.java

```diff
@@ -172,14 +172,16 @@ public void dispose() {
   private static final Flag<?>[] ALL_FLAGS = {
     Flags.help,
     Flags.version,
-    GapiPaths.gapidPath,
+    Follower.logFollowRequests,
```

Comment:
Done. I do hate flag lists that aren't alphabetically ordered though (not that I would expect declaration order to affect presentation).